### ISSUE TITLE
Use axe-core to set up automated accessibility testing

### ISSUE THREAD

**Issue by codeAbul** [tatianamac/selfdefined#48]
I think axe-core( https://github.com/dequelabs/axe-core ) is a lovely little tool we could incorporate into our dev workflow to catch any accessibility violations at the earliest. Would love to hear thoughts on this

**Comment by tatianamac** [comment]
@ovlb @dengeist I would love to hear your thoughts on this specifically!

**Comment by ovlb** [comment]
I would be careful to say «any violations» as automatic testing can only find a subset of violations/problems with the accessibility of a site. Besides from nitpicking though, I think it makes sense to build up a testing/linting infrastructure, and axe should be a part of it.

**Comment by dengeist** [comment]
Agreed! Linting is mainly here to help us catch the low-hanging fruit, but that's fine; getting it in place will allow us to better claim responsibility for the rest of the app's accessibility.

**Comment by tatianamac** [comment]
Great, thank you, @dengeist and @ovlb . I'm going to add a separate issue that includes "Set up testing/linting infrastructure/workflow."

**Comment by codeAbul** [comment]
@tatianamac @ovlb @dengeist thanks for the feedback! Yes @ovlb automated accessibility testing is only the first step and definitely a good starting point!

**Comment by greatislander** [comment]
@tatianamac I think this actually needs to be re-opened as @codeAbul was going to tackle it once the other CI had been set up.

**Comment by ovlb** [comment]
We #74 to keep track of the single parts, so maybe having this closed gets rid of some duplication.
